### PR TITLE
Samples readme

### PR DESCRIPTION
**Pull request by a-tylenda**
Testing ts samples after new npm release.

### CODE REVIEW

File: samples/stack-overflow/stack-overflow.ts [hunk 1]
```diff
@@ -28,11 +28,18 @@ const ONE_MINUTE = 60000;
 async function diff(interval: number, key?: string) {
 
     const url = key ? `${BASE_URL}${TAGS_URL}&key=${key}` : `${BASE_URL}${TAGS_URL}`;
+
+    console.log("-------URL: ", url);
```
Comment (alicja-gruzdz) [inline samples/stack-overflow/stack-overflow.ts:32]: Leftover debug :)

File: samples/stack-overflow/stack-overflow.ts [hunk 2]
```diff
@@ -28,11 +28,18 @@ const ONE_MINUTE = 60000;
...
+
     const before = (await axios.get(url)).data.items;
+
+    console.log("--------Before--------", before);
```
Comment (alicja-gruzdz) [inline samples/stack-overflow/stack-overflow.ts:36]: de:bug:

File: samples/stack-overflow/stack-overflow.ts [hunk 3]
```diff
@@ -28,11 +28,18 @@ const ONE_MINUTE = 60000;
...
   
     await sleep(interval * ONE_MINUTE);
   
     const after = (await axios.get(url)).data.items;
+
+    console.log("--------after--------", after);
```
Comment (alicja-gruzdz) [inline samples/stack-overflow/stack-overflow.ts:42]: de:bug:

File: samples/transform-string-stream/README.md [hunk 4]
```diff
@@ -36,13 +38,16 @@ si seq start <sequence-id> Hello Bye
 
 # See output of instance process
 si inst output <instance-id>
+```
 
-# In another terminal send text to instance input steam
+**The third terminal**
+```bash
+# Send file to instance input steam
```
Comment (alicja-gruzdz) [inline samples/transform-string-stream/README.md:45]:
```suggestion
# Send file to the instance input steam
```

File: samples/stack-overflow/index.ts [hunk 5]
```diff
@@ -12,6 +12,8 @@ const app: ReadableApp<string> = async function (_stream: any, interval: number,
         if (Object.keys(result.diff).length > 0) outputStream.write(result);
     }, interval * ONE_MINUTE);
 
+    (outputStream as SynchronousStreamable<any>).contentType = "application/x-ndjson";
```
Comment (alicja-gruzdz) [inline samples/stack-overflow/index.ts:15]: What do you think if we added the comment above this line explaining what the content type setting is for?
e.g.
`# If you wish you operate the output stream in "object mode", please specify the content type of the returned stream to be "application/x-ndjson". You can implement by using SynchronousStreamable type from "@scramjet/types".`

**Review by alicja-gruzdz — APPROVED** [review]
Great fruits of tedious debug & testing :)

File: samples/stack-overflow/index.ts (hunk 5 above)
Comment (a-tylenda) [reply samples/stack-overflow/index.ts:16]: Thank you Ala! I totally forgot about it. Comment added as suggested :)

**Review by agne6ka — APPROVED** [review]

**Comment by agne6ka** [comment]
@a-tylenda I'm thinking about [what is scramjet section](https://github.com/scramjetorg/scramjet-cloud-docs#1-what-is-scramjet-transform-hub-sth). Right now it seems like a wall of text.

Maybe we can make a short intro to what is STH and then add a FAQ similar [to API - I mean appearance](https://github.com/scramjetorg/transform-hub/blob/release/0.13/docs/interfaces/API-reference.md).

**Comment by a-tylenda** [comment]
> @a-tylenda I'm thinking about [what is scramjet section](https://github.com/scramjetorg/scramjet-cloud-docs#1-what-is-scramjet-transform-hub-sth). Right now it seems like a wall of text.
> 
> Maybe we can make a short intro to what is STH and then add a FAQ similar [to API - I mean appearance](https://github.com/scramjetorg/transform-hub/blob/release/0.13/docs/interfaces/API-reference.md).

great idea! I will do so :)

**Pull request merged**